client/components/card/index.jsx

```diff
@@ -25,38 +37,26 @@ class Card extends Component {
 	};
 
 	render() {
-		const { children, compact, highlight, href, onClick, tagName, target } = this.props;
-
-		const highlightClass = highlight ? 'is-' + highlight : false;
-
-		const className = classnames(
-			'card',
-			this.props.className,
-			{
-				'is-card-link': !! href,
-				'is-clickable': !! onClick,
-				'is-compact': compact,
-				'is-highlight': highlightClass,
-			},
-			highlightClass
-		);
-
-		const omitProps = [ 'compact', 'highlight', 'tagName' ];
+		const {
+			children,
+			compact, // eslint-disable-line no-unused-vars
+			highlight, // eslint-disable-line no-unused-vars
+			highlightClass, // eslint-disable-line no-unused-vars
+			tagName: TagName,
+			href,
+			target,
+			...props
+		} = this.props;
 
-		let linkIndicator;
-		if ( href ) {
-			linkIndicator = (
+		return href ? (
+			<a { ...props } href={ href } target={ target } className={ getClassName( this.props ) }>
```

Comment:
For folks reading, I tend to prefer a whitelist for clarity over `{ ...props }`, but this is a special case where we want to pass through potentially any valid html5 link tag prop.